fixes map sizing issues and table pagination

designmatty opened the pull request:
## Overview

- fixes #328 
- fixes #314 
- fixes #313 
- added a class to contributor meta-data (in the map/sidebar layout) which will force a line break when the string is longer than the containing sidebar.

## Demo

<img width="854" alt="Screen Shot 2019-03-26 at 9 27 54 AM" src="https://user-images.githubusercontent.com/1928955/55000807-b2d6ce00-4fa9-11e9-878b-4881b923c628.png">
<img width="904" alt="Screen Shot 2019-03-26 at 9 28 14 AM" src="https://user-images.githubusercontent.com/1928955/55000808-b2d6ce00-4fa9-11e9-83d6-c96a16ef7c45.png">
<img width="927" alt="Screen Shot 2019-03-26 at 9 28 22 AM" src="https://user-images.githubusercontent.com/1928955/55000809-b36f6480-4fa9-11e9-9b29-df5c801b4b1e.png">

## Testing Instructions

* view a facility and check it's static map. 
  - Should no longer be warped.
* view a facility contributor list table, resize screen and scroll table.
  - Pagination should not scroll left or right.

kellyi commented:
Heads up that there are some linter errors preventing CI from building this branch:

```
/usr/local/src/src/components/FacilityListItemsTable.jsx
    9:8  error  'TableRow' is defined but never used                     no-unused-vars
  194:9  error  Expected indentation of 12 space characters but found 8  react/jsx-indent
  213:1  error  Expected indentation of 12 spaces but found 8            indent
  213:9  error  Expected indentation of 12 space characters but found 8  react/jsx-indent

✖ 4 problems (4 errors, 0 warnings)
  3 errors and 0 warnings potentially fixable with the `--fix` option.
```

designmatty commented:
Sorry about that, I always forget this project has a linter.

jwalgran commented on src/app/src/components/FacilityListItemsTable.jsx:
```diff
@@ -192,29 +189,29 @@ function FacilityListItemsTable({
-                </Table>
-            </div>
-        </Paper>
+        <>
```
Is this empty tag intentional/required?

designmatty replied:
unintentional

designmatty replied:
This shouldnt be required. I'll remove it.

designmatty commented:
I rebased and updated some styling to that message. I was able to keep the message within the area it was originally placed.

<img width="949" alt="Screen Shot 2019-03-27 at 9 05 24 AM" src="https://user-images.githubusercontent.com/1928955/55078254-1e836e80-5070-11e9-984b-6d96cf32cb64.png">
